python/grass/jupyter/display.py

```diff
@@ -0,0 +1,101 @@
+
+# MODULE:    grass.jupyter.display
+#
+# AUTHOR(S): Caitlin Haedrich <caitlin DOT haedrich AT gmail>
+#
+# PURPOSE:   This module contains functions for non-interactive display
+#            in Jupyter Notebooks
+#
+# COPYRIGHT: (C) 2021 Caitlin Haedrich, and by the GRASS Development Team
+#
+#           This program is free software under the GNU General Public
+#           License (>=v2). Read the file COPYING that comes with GRASS
+#           for details.
+
+import os
+import folium
```

Comment:
Probably won't be part of this PR at the end and not a big deal in Binder, but in general, this will need to be a lazy import, only in the function which needs it unless you make the "interactive" part a separate module and class which may or may not happen in the future. As in the case of `from IPython.display import Image`.